### PR TITLE
COL-839, loadOtherUsers() after loadProfile(); hide oi-select arrow

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/COL-839

Two small adjustments:
* Because we're not using `multiple` in `oi-select` we get an unwanted arrow in input element. This is not the first oi-select customization in our code...
* The previous placement of `$scope.otherUsers` assignment could have been a problem. Let's avoid the risk.

### CODE REVIEW

File: public/app/dashboard/profileController.js [hunk 1]
```diff
@@ -139,24 +139,28 @@
       $scope.user.hashtags = ['#badminton', '#bridge', '#break-dancing'];
     };
 
+    var loadOtherUsers = function(user) {
+      // The dropdown of 'search for other users' needs the following
+      // list of otherUsers. We exclude user.id of current profile view.
+      userFactory.getAllUsers().then(function(response) {
+        $scope.otherUsers = _.reject(response.data, function(otherUser) {
```
Comment (paulkerschen) [inline public/app/dashboard/profileController.js]: @johncrossman I believe Lodash supports a one-liner shorthand here:  
```
_.reject(response.data, {'id': user.id});
```

Comment (johncrossman) [inline public/app/dashboard/profileController.js]: Nifty! Amending now.

Comment (johncrossman) [inline public/app/dashboard/profileController.js]: The Lodash shorthand was tested and works. Amendment is in.

**Comment by paulkerschen** [comment]
Waiting for T to clean out the lint...

**Pull request merged**